src/parsers/string.rs

```diff
@@ -314,19 +314,33 @@ mod tests {
 
     #[test]
     fn length_can_contain_leading_zeros() {
-        assert_eq!(bencode_to_json_unchecked(b"00:"), r#""""#.to_string());
+        assert_eq!(
+            bencode_to_json_unchecked(b"00:"),
+            r#""<string></string>""#.to_string()
+        );
```

Comment:
Hi @magecnion I have not thought about this case. Empty byte sequences could be mapped to `hex` of `string` tags. However, I prefer what you did. Since tags are mandatory and JSON uses UTF-8 I prefer to use the tag `<string>` for empty byte sequences.